README.md

```diff
@@ -9,7 +9,10 @@
 
 # Using the Yam Uma Synths SDK
 
-### **Official assets:** [File](https://github.com/yam-finance/synths-sdk/blob/master/src/assets.json) and [Raw file](https://raw.githubusercontent.com/yam-finance/synths-sdk/master/src/assets.json)
+### \*\*Official
```

Comment:
Why do you do this?
 
 Have a look at how this is rendered - it looks like this is not what you want to achieve.

Reply:
IDE sillyness. Reverted with b1da1cc